### PR TITLE
Relocate proxy TLS certificate into proper namespace path of secret store

### PR DESCRIPTION
**Pull request by tingyuz**
Fix for #1931

Starting in Geneva the path to store secrets in secretstore has been standardized, it is necessary to update the location of proxy TLS cert to follow the rule

Signed-off-by: Tingyu Zeng <Tingyu.Zeng@rsa.com>

### CODE REVIEW

**Comment by bnevis-i** [comment]
Actually, there is no "edgex-kong" service key.  It should be relocated to
`v1/secret/edgex/edgex-security-proxy-setup/kong-tls`

In parallel with this, the token config file, `cmd/security-file-token-provider/res/token-config.json`, should have the exception path for `secret/edgex/pki/tls/edgex-kong` removed in `edgex-security-proxy-setup`  (since it will use the standard location now) and `edgex-security-secretstore-setup` path should be updated to above for documentation purposes (note: potential future improvement to deprivilege itself and use a token that it generates for itself for this purpose).

**Comment by tingyuz** [comment]
@bnevis-i will update edgex-security-secretstore-setup in the token-config.json file. 
Since we only have one path (secret/edgex/pki/tls/edgex-kong) under edgex-security-proxy-setup in the token-config.json file, can we delete the custom_policy section of edgex-security-proxy-setup at all?

**Comment by bnevis-i** [comment]
> @bnevis-i will update edgex-security-secretstore-setup in the token-config.json file.
> Since we only have one path (secret/edgex/pki/tls/edgex-kong) under edgex-security-proxy-setup in the token-config.json file, can we delete the custom_policy section of edgex-security-proxy-setup at all?

Yes.  It should change to:
```
  "edgex-security-proxy-setup": {
    "edgex_use_defaults": true
  },
```

File: cmd/security-proxy-setup/res/configuration.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ WhiteList = "admin"
 Server = "localhost"
 Port = 8200
 HealthCheckPath = "v1/sys/health"
-CertPath = "v1/secret/edgex/pki/tls/edgex-kong"
+CertPath = "secret/edgex/edgex-security-proxy-setup/kong-tls"
```
Comment (bnevis-i) [inline cmd/security-proxy-setup/res/configuration.toml]:
```suggestion
CertPath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
```

File: cmd/security-proxy-setup/res/docker/configuration.toml [hunk 2]
```diff
@@ -45,7 +45,7 @@ WhiteList = "admin"
 Server = "edgex-vault"
 Port = 8200
 HealthCheckPath = "v1/sys/health"
-CertPath = "v1/secret/edgex/pki/tls/edgex-kong"
+CertPath = "secret/edgex/edgex-security-proxy-setup/kong-tls"
```
Comment (bnevis-i) [inline cmd/security-proxy-setup/res/docker/configuration.toml]:
```suggestion
CertPath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
```

**Comment by bnevis-i** [comment]
Validation failed due to typo in CertPath.

File: cmd/security-proxy-setup/res/configuration.toml (hunk 1 above)
Comment (tingyuz) [reply cmd/security-proxy-setup/res/configuration.toml]: Fixed.

Comment (tingyuz) [reply cmd/security-proxy-setup/res/configuration.toml]: Fixed.

File: cmd/security-proxy-setup/res/docker/configuration.toml (hunk 2 above)
Comment (tingyuz) [reply cmd/security-proxy-setup/res/docker/configuration.toml]: Fixed.

**Comment by tingyuz** [comment]
recheck

**Pull request merged**